openslides/motions/serializers.py

```diff
@@ -219,7 +223,7 @@ def create(self, validated_data):
         motion.reason = validated_data.get('reason', '')
         motion.identifier = validated_data.get('identifier')
         motion.category = validated_data.get('category')
-        motion.reset_state(validated_data.get('workflow', int(config['motions_workflow'])))
+        motion.reset_state(validated_data.get('workflow_id'))
```

Comment:
Why not using the config if the client does not send an workflow_id? The field is not required so we need a fallback here.